relay-server/src/services/projects/cache/project.rs

```diff
@@ -89,7 +89,7 @@ impl<'a> Project<'a> {
         // If we can extract spans from the event, we want to try and count the number of nested
         // spans to correctly emit negative outcomes in case the transaction itself is dropped.
         relay_statsd::metric!(timer(RelayTimers::CheckNestedSpans), {
```

Comment:
I think this should be moved into `ensure_span_count` now, just the branch which actually parses the contents.